### PR TITLE
[FIX] Fixed method call to get closest branch name

### PR DESCRIPTION
**Pull request by PCatinean**

### CODE REVIEW

File: runbot_pylint/models/runbot_build.py [hunk 1]
```diff
@@ -91,8 +91,8 @@ def get_repo_branch_name(self):
         for build in self:
             hint_branches = set()
             for extra_repo in build.repo_id.dependency_ids:
-                closest_name = build.get_closest_branch_name(
-                    extra_repo.id, hint_branches)
+                repo_id, closest_name = build._get_closest_branch_name(
+                    extra_repo.id)
```
Comment (gurneyalex) [inline runbot_pylint/models/runbot_build.py]: which branch of odoo-extra are you using as reference for this method rename?

Comment (PCatinean) [inline runbot_pylint/models/runbot_build.py]: The latest version of master, the function call is the same as: 
https://github.com/odoo/odoo-extra/blob/master/runbot/runbot.py#L732

**Comment by pedrobaeza** [comment]
Waiting for the answer to review easily...

**Comment by gurneyalex** [comment]
:+1:

File: runbot_pylint/models/runbot_build.py (hunk 1 above)
Comment (pedrobaeza) [inline runbot_pylint/models/runbot_build.py]: If you're not using repo_id, make `closest_name = build._get_closest_branch_name(extra_repo.id)[1]`

**Comment by pedrobaeza** [comment]
:+1: Thanks, merging...

**Pull request merged**